lib/core/utils/valid-langs.js

```diff
@@ -1,4 +1,71 @@
+
+/**
+ * Determine if a string is a valid language code
+ * @method isValidLang
+ * @memberof axe.utils
+ * @param {String} lang String to test if a valid language code
+ * @returns {Boolean}
+ */
+function isValidLang(lang) {
+	let array = langs;
+
+	// padEnd is not supported in IE11
+	while (lang.length < 3) {
+		lang += '`';
+	}
+	const chars = lang.split('');
+
+	for (let i = 0; i <= chars.length - 1; i++) {
+		const index = chars[i].charCodeAt(0) - 96;
+		array = array[index];
```

Comment:
Seems like we could just do this:

```suggestion
	for (let i = 0; i <= lang.length - 1; i++) {
		const index = lang.charCodeAt(i) - 96;
		array = array[index];
```